src/ape/api/providers.py

```diff
@@ -26,6 +26,10 @@ class TransactionType(Enum):
 
 @abstractdataclass
 class TransactionAPI:
+    """
+    An API class representing a Transaction API.
```

Comment:
A good test for docs like this is asking "does this tell me something I don't already know from reading the code?"

Try to expand this by asking "what is this used for?" and "why would I want to use this?"

Reply:
Yeah, maybe something like

```
"""
An API class representing a transaction.
Ecosystem plugins implement one or more of transaction APIs depending on which schema it allows,
such as typed-transactions from EIP-1559.
"""
```